lib/crawler_web/controllers/home_controller.ex

```diff
@@ -4,10 +4,10 @@ defmodule CrawlerWeb.HomeController do
   alias Crawler.Keyword.Keywords
   alias Crawler.Keyword.Schemas.KeywordCSVFile
 
-  def index(conn, _params) do
+  def index(conn, params) do
```

Comment:
I think think we should update the test for this controller 🤔 aside from the system test @vnntsu what do you think?